internal/docker/container.go

```diff
@@ -309,6 +313,11 @@ func (r *ContainerRunner) collectResults(results chan result, expected int) bool
 	return passed
 }
 
+func getJobID(url string) string {
```

Comment:
I think this one deserves a bit more info. Especially since it's a package level function. From the signature alone, it's unclear what type of URL is expected for this function to perform its job.

Reply:
updating